nyxtools/vector.py

```diff
@@ -232,28 +233,50 @@ def prepare_move(
         self.timeout = 5 * estimated_total_time_ms / 1000.0
         self.ready = True
 
-    def start_move(self):
+    def move(self):
         if not self.ready:
             raise Exception("Must execute prepare_move command before move is allowed.")
         # Start actual motion
         self.calc_only.put(False)
 
-        def start_callback(value, old_value):
-            if old_value == 0 and value == 1:
+        # Note: 'kwargs' are needed here as extra information is passed to the callback, such as:
+        # kwargs: {'timestamp': 1638904545.824989,
+        #          'status': <AlarmStatus.NO_ALARM: 0>,
+        #          'severity': <AlarmSeverity.NO_ALARM: 0>,
+        #          'enum_strs': ('No', 'Yes'),
+        #          'sub_type': 'value',
+        #          'obj': EpicsSignalRO(read_pv='XF:19IDC-ES{Gon:1-Vec}Sts:Running-Sts',
+        #                               name='vector_active', parent='vector', value=0,
+        #                               timestamp=1638904545.824989, auto_monitor=False,
+        #                               string=False)}
+        def start_callback(value, old_value, **kwargs):
+            print(f"{time.ctime()}: {old_value} -> {value}")
+            if old_value == "Backup" and value == "Acquiring":
+                print(f"{time.ctime()}: Successfully changed {old_value} -> {value}")
                 return True
             else:
+                print(f"{time.ctime()}: changing {old_value} -> {value}...")
                 return False
 
-        run_status = SubscriptionStatus(self.running, start_callback, run=False)
+        run_status = SubscriptionStatus(self.state, start_callback, run=True)
+        print(f"{time.ctime()}: subscribed to {self.active.name}")
+
+        time.sleep(1.0)
```

Comment:
Why did we need this 1s timeout here?

Reply:
I can't think of a single good reason, which makes me think it's an artifact of the original code. I'll remove it.